src/app/main.cpp

```diff
@@ -245,11 +247,26 @@ static bool render(GLFWwindow* window) {
     position_layer(sw_stats, params, window_size);
     render_imgui(sw_stats, params, window_size, true);
     overlay_end_frame();
-    glfwSetWindowSize(window, 1280, 800);
+    glfwSetWindowSize(window, screen_width, screen_height);
     ImGui::EndFrame();
     return last_window_size.x != window_size.x || last_window_size.y != window_size.y;
 }
 
+static void updateScreenInfo(){
+    GLFWmonitor* pMonitor = glfwGetPrimaryMonitor();
+    const GLFWvidmode * mode = glfwGetVideoMode(pMonitor);
+    screen_width = mode->width;
+    screen_height = mode->height;
```

Comment:
Will this work correctly when the primary (and only) monitor gets disconnected? Say when your dog pulls the cable for a moment, because they want you to play with them :dog2:

Reply:
I will update a new file to resolve those problems